packages/imu_driver/include/hardware_test_imu/hardware_test_imu.py

```diff
@@ -0,0 +1,66 @@
+import rospy
+
+from dt_duckiebot_hardware_tests import HardwareTest, HardwareTestJsonParamType
+
+
+class HardwareTestIMU(HardwareTest):
+    def __init__(self) -> None:
+        super().__init__()
+
+    def test_id(self) -> str:
+        return f"IMU"
+
+    def test_description_preparation(self) -> str:
+        return self.html_util_ul(
+            [
+                "Put your Duckiebot in its normal orientation, on a surface (e.g. desk).",
+                "Make sure you can reach and grab the robot when the test starts.",
+            ]
+        )
+
+    def test_description_running(self) -> str:
+        return self.html_util_ul(
+            [
```

Comment:
Remove this to match the format of other tests - working it into the expectations section